### ISSUE TITLE
World Malaria Day 2018

### ISSUE THREAD

**Issue by mattodd** [OpenSourceMalaria/OSM_To_Do_List#562]
...is on [April 25th](https://www.mmv.org/newsroom/events/world-malaria-day-2018) (a big Aussie public holiday as it happens).

**What can we do as a community to mark this occasion? Calling for ideas.**

In the past we have pushed social media messages to raise awareness, which is fine and valuable.

I wonder, though, if we could do something more concrete. A very good thing we could do is to aim to ship the next round of compounds to Dundee on that date (actually 24th). @edwintse @david1597 @maratsydney @mbhebhe - what say you? Reasonable?

Another thing we can do that is substantive is summarise the other molecules, not in the shipment, that are currently high value targets and bring peoples' attention to the ongoing synthetic work.

I wonder if @mcoster and @holeung and all others reading this Issue would be willing to prepare something written and short that describes current work relevant to OSM Series?

We could collate everything into a neat new Issue, and link to it on the day.

I will use the event as an excuse to (finally) send out the first newsletter (#547). This was delayed by a trivial thing I was waiting on, but which doesn't have a simple resolution, so I'll go ahead without it.

**Comment by david1597** [comment]
Yes - that'd be a great point at which to send the next shipment to Dundee. There should be 15-20 compounds between us.

Also great opportunity for distributing the first newsletter. Good stuff.

**Comment by cdsouthan** [comment]
As just tweeted, update/clean up the master sheet and surface it on figshare?

**Comment by mattodd** [comment]
That would be awesome of you, @cdsouthan.

**Comment by holeung** [comment]
Lovely! I'll write something up. Maybe we can do a press release?

**Comment by cdsouthan** [comment]
@david1597 given I could get a least a few front runners into PubChem for WMD but to assist could  you list out 
a) how many sub-100 nM hits do we have in S4?
b) ditto sub-500 nM?

**Comment by mattodd** [comment]
...but @cdsouthan can you filter on the average potency in the Master List?

**Comment by cdsouthan** [comment]
1) @david1597 @edwintse  Thanks, the sub 100s will do fine for now so don't bother with any more.   I'll need full strings so perhaps one of you could just highlight in green the corresponding master-sheet rows and I can take it from there,

**Comment by edwintse** [comment]
@cdsouthan I think the quickest thing to do would be for you to search for the OSM numbers (they'll be in numerical order) in the master sheet to find the corresponding strings.

**Comment by cdsouthan** [comment]
Fine,  but just  for the record I was not trying to be lazy,  its just that every extra manual operation has an error probability, especially any I have to do.  Notwithstanding,   I will try my damnedest to locate the correct string sets for all ll  :)

**Comment by david1597** [comment]
Apologies for deleting Ed's comment. The table below contains the strings. 

OSM ID | Internal   ID | MMV ID | SMILES | InChI | IChI   Key |   | IC50
-- | -- | -- | -- | -- | -- | -- | --
OSM-S-377 | INHERITED | MMV670652 | FC(C=C1)=C(F)C=C1C(OC(F)F)COC2=CN=CC3=NN=C(C4=CC=C(C#N)C=C4)N32 | InChI=1S/C21H13F4N5O2/c22-15-6-5-14(7-16(15)23)17(32-21(24)25)11-31-19-10-27-9-18-28-29-20(30(18)19)13-3-1-12(8-26)2-4-13/h1-7,9-10,17,21H,11H2 | LHSRVKYAEUESTK-UHFFFAOYSA-N | 16.7
OSM-S-381 | INHERITED | MMV670947 | FC(F)OC(C=C1)=CC=C1C2=NN=C3C=NC=C(OCC(CO)C4=CC(F)=C(F)C=C4)N32 | InChI=1S/C21H16F4N4O3/c22-16-6-3-13(7-17(16)23)14(10-30)11-31-19-9-26-8-18-27-28-20(29(18)19)12-1-4-15(5-2-12)32-21(24)25/h1-9,14,21,30H,10-11H2 | MKSKANAVVSFYNL-UHFFFAOYSA-N | 24.3
  | INHERITED | MMV670945 | FC(F)OC(C=C1)=CC=C1C2=NN=C3C=NC(OCCC4=CC=C(F)C(F)=C4)=CN32 | InChI=1S/C20H14F4N4O2/c21-15-6-1-12(9-16(15)22)7-8-29-18-11-28-17(10-25-18)26-27-19(28)13-2-4-14(5-3-13)30-20(23)24/h1-6,9-11,20H,7-8H2 | PHGWNHSKOAAQGR-UHFFFAOYSA-N | 33.8
OSM-S-218 | AEW   313-1 | MMV669844/MMV897709 | FC(C=C1)=C(F)C=C1C(OC)COC2=CN=CC3=NN=C(C4=CC=C(C#N)C=C4)N32 | InChI=1S/C21H15F2N5O2/c1-29-18(15-6-7-16(22)17(23)8-15)12-30-20-11-25-10-19-26-27-21(28(19)20)14-4-2-13(9-24)3-5-14/h2-8,10-11,18H,12H2,1H3/t18-/m0/s1 | 40
  | INHERITED | MMV670437 | FC(F)OC(C=C1)=CC=C1C2=NN=C3C=NC=C(OC[C@H](N(C)C)C4=CC(F)=C(F)C=C4)N32 | InChI=1S/C22H19F4N5O2/c1-30(2)18(14-5-8-16(23)17(24)9-14)12-32-20-11-27-10-19-28-29-21(31(19)20)13-3-6-15(7-4-13)33-22(25)26/h3-11,18,22H,12H2,1-2H3/t18-/m0/s1 | PMIWBIXSAYKRGF-SFHVURJKSA-N |   | 44.1
OSM-S-418 | EGT   257-1 | MMV1576784 | FC(F)OC(C=C1)=CC=C1C2=NN=C3C=NC=C(OCC/C4=C/BBBBBBBBBB4)N32 | InChI=1S/C16H22B10F2N4O2/c27-16(28)34-12-3-1-10(2-4-12)15-31-30-13-8-29-9-14(32(13)15)33-6-5-11-7-17-19-21-23-25-26-24-22-20-18-11/h1-4,7-9,16-26H,5-6H2/b11-7- | AGPVLTOPROUGPQ-XFFZJAGNSA-N |   | 50
  | INHERITED | MMV672936 | FC(F)OC(C=C1)=CC=C1C2=NN=C3C=NC=C(OCC(F)C4=CC=C(F)C(F)=C4)N32 | InChI=1S/C20H13F5N4O2/c21-14-6-3-12(7-15(14)22)16(23)10-30-18-9-26-8-17-27-28-19(29(17)18)11-1-4-13(5-2-11)31-20(24)25/h1-9,16,20H,10H2 | LWSXCRQHVHDFGX-UHFFFAOYSA-N | 65.4
OSM-S-272 | AEW   302-1 | MMV639565 | FC1=C(F)C=CC(CCOC2=CN=CC3=NN=C(C4=CC=C(Cl)C=C4)N32)=C1 | InChI=1S/C19H13ClF2N4O/c20-14-4-2-13(3-5-14)19-25-24-17-10-23-11-18(26(17)19)27-8-7-12-1-6-15(21)16(22)9-12/h1-6,9-11H,7-8H2 | PMHUSEXABGDNGS-UHFFFAOYSA-N | 68.8
OSM-S-525 | MK035-1 | MMV1579341 | FC1=CC(CCOC2=CN=CC3=NN=C(C4=CC=C5C(NC=C5)=C4)N32)=CC=C1F | InChI=1S/C21H15F2N5O/c22-16-4-1-13(9-17(16)23)6-8-29-20-12-24-11-19-26-27-21(28(19)20)15-3-2-14-5-7-25-18(14)10-15/h1-5,7,9-12,25H,6,8H2 | VHFREJHLTSVZGA-UHFFFAOYSA-N |   | 86
OSM-S-390 | EGT   171-1 | MMV672687 | FC(OC1=CC=C(C2=NN=C3C=NC=C(N32)OCC(C4=CC(F)=C(C=C4)F)O)C=C1)F | InChI=1S/C20H14F4N4O3/c21-14-6-3-12(7-15(14)22)16(29)10-30-18-9-25-8-17-26-27-19(28(17)18)11-1-4-13(5-2-11)31-20(23)24/h1-9,16,20,29H,10H2 | BXEWPSCIVPVASC-UHFFFAOYSA-N | 99

**Comment by cdsouthan** [comment]
Just the job :)

**Comment by cdsouthan** [comment]
@david1597  please resolve 

OSM-S-218 | AEW 313-1 | MMV669844/MMV897709
-- | -- | --

 an include an InChIKey

**Comment by david1597** [comment]
@cdsouthan 

I've edited my comment above to add the missing InChIKey.

The double MMV assignment relates to OpenSourceMalaria/Series4#35 and #470. OSM-S-218 is the compound which was resolved into it's enantiomers. I'm waiting to hear back from UCSD on the liver stage results (OpenSourceMalaria/Series4#35), and once that is clear we can fix up any other problems relating to these compounds.

**Comment by mattodd** [comment]
OK all, I've started the relevant GHI at #564 

Please feel free to add a few lines about one thing you're working on. No need to make it comprehensive or pretty. Short and simple is good. Please try to include a link out to somewhere your work is (another Issue, for example) and, if you're able, add a photo to keep the page light. World Malaria Day is on Wednesday, so I'll probably start sharing #564 around Wed afternoon in Australia, and share a few more times as Europe and the US wakes up. So you have about 48 hours from the time of this message.

I'll keep this issue open for now, since we're still discussing a cheminformatics issue that has snuck in I think @cdsouthan ?

**Comment by cdsouthan** [comment]
1) have done
2) Most of front-runners are now up at  https://www.ncbi.nlm.nih.gov/pccompound?term=%22TW2Informatics%22%5BSourceName%5D&cmd=DetailsSearch
there was an issue with one SMILES string that I need to check out.  Note they are not yet expanded to BioAssay entries but I intent to do this in due course
3) As I think was intended I will put a copy of the master-sheet on my figshare tomorrow
